Library/Homebrew/tap.rb

```diff
@@ -133,6 +135,18 @@ def install(options = {})
     end
   end
 
+  def link
+    if (path/"man").exist?
+      (path/"man").find do |src|
+        next if src.directory?
+        dst = HOMEBREW_PREFIX/src.relative_path_from(path)
+        FileUtils.mkdir_p dst.dirname
+        dst.delete if dst.exist?
```

Comment:
This doesn't meet our policy on not deleting users files without consent. We could print warning message similar in `brew link`.